nixos/modules/services/web-apps/rimgo.nix

```
@@ -0,0 +1,55 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.services.rimgo;
+in {
+  options.services.rimgo = {
+    enable = mkEnableOption "rimgo";
+    package = mkPackageOption pkgs "rimgo" {};
+    settings = mkOption {
+      type = with types; attrsOf str;
+      default = {};
+      example = literalExpression ''
+        {
+          FORCE_WEBP = "1";
+        }
+      '';
+      description = ''
+        Settings for rimgo, see [here](https://rimgo.codeberg.page/docs/usage/configuration/) for supported options.
+      '';
+    };
+    port = mkOption {
+      type = types.port;
+      default = 3000;
+      example = 69420;
+      description = "The port to use";
+    };
+    address = mkOption {
+      type = types.str;
+      default = "127.0.0.1";
+      description = "The address to listen on";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    services.rimgo.settings = {
+      PORT = mkDefault (toString cfg.port);
+      ADDRESS = mkDefault cfg.address;
+    };
+    systemd.services.rimgo = {
+      description = "Rimgo";
+      wantedBy = ["default.target"];
```

```suggestion
      wantedBy = [ "multi-user.target" ];
      after = [ "network.target" ];
```
I don't think rimgo is important enough to warrant `default.target`. And it certainly needs network. The `partOf` can be removed entirely, I think (I don't know if, or how, it is possible to refer to multiple lines in a review here).